src/animation/AnimationAction.js

```diff
@@ -473,7 +473,7 @@ AnimationAction.prototype = {
 			if ( loopCount === -1 ) {
 				// just started
 
-				this.loopCount = 0;
+				this._loopCount = 0;
```

Comment:
More explanation about context of this.

There is a private variable call _loopCount at line 51:

```javascript 
this._loopCount = -1;
```

and a local loopCount variable inside _updateTime method that take the _loopCount value at initialization. This could be provoc undefined behavior, or to contrary fix a not detected bug. (I don't analyse in deep this code yet). But the private _loopCount variable is used only 5 times, and just in the _updateTime and reset (to his original value of -1) methods... So...